src/main/java/com/google/devtools/build/lib/packages/StarlarkNativeModule.java

```diff
@@ -372,40 +719,26 @@ private static Object starlarkifyValue(Mutability mu, Object val, Package pkg)
       }
       return m.build(mu);
     }
-    if (val.getClass().isAnonymousClass()) {
-      // Computed defaults. They will be represented as
-      // "deprecation": com.google.devtools.build.lib.analysis.BaseRuleClasses$2@6960884a,
-      // Filter them until we invent something more clever.
-      return null;
-    }
 
-    if (val instanceof License) {
-      // License is deprecated as a Starlark type, so omit this type from Starlark values
-      // to avoid exposing these objects, even though they are technically StarlarkValue.
-      return null;
+    if (val instanceof BuildType.SelectorList) {
+      List<Object> selectors = new ArrayList<>();
+      for (BuildType.Selector<?> selector : ((BuildType.SelectorList<?>) val).getSelectors()) {
+        selectors.add(
+            new SelectorValue(
+                ((Map<?, ?>) starlarkifyValue(mu, selector.getEntries(), pkg)),
+                selector.getNoMatchError()));
+      }
+      try {
+        return SelectorList.of(selectors);
+      } catch (EvalException e) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**: While wrapping the caught exception into a custom one, information about the caught exception is being lost, including information about the stack trace of the exception.

**Fix**: If the caught exception object does not contain sensitive information, consider passing it as the &#34;rootCause&#34; or inner exception parameter to the constructor of the new exception before throwing the new exception. (Note that not all exception constructors support inner exceptions. Use a wrapper exception that supports inner exceptions.) 
[Learn more](https://www.ibm.com/support/pages/best-practice-catching-and-re-throwing-java-exceptions)